### DISCUSSION TITLE
Build size

### DISCUSSION THREAD

**Discussion by TbirdDuncan** [preactjs/preact-cli#1655] [Q&A] [answered]
Hi, I'm very curious as to how the build files work in conjuction with one another. Because atm there seems to be no benefit to using preact over just react, the file sizes add up to about the same. Thank you

**Comment by rschristian** [comment, marked as answer]
Preact vs React is likely a minor portion of your overall app bundle.

Depending on how you build your React app, that can also have fundamental differences in bundle output to what Preact-CLI does.
